library/x509_crt.c

```diff
@@ -1320,7 +1325,15 @@ int mbedtls_x509_crt_parse_path( mbedtls_x509_crt *chain, const char *path )
         }
 
         w_ret = mbedtls_x509_crt_parse_file( chain, filename );
-        if( w_ret < 0 )
+        /*
+         * Quit parsing on a feature not suppoprted by HW error.
+         */
+        if( w_ret == MBEDTLS_ERR_PLATFORM_FEATURE_UNSUPPORTED )
```

Comment:
Because we need to know if the error was due to a feature unsupported error or any other error.
This function returns the number of certificates failed to parse. In a case that one of the certificates failed because of unsupported feature, we need a way to identify, and skip the tests. Note this is still a failure.